.cirrus.yml

```diff
@@ -273,6 +283,10 @@ success_task:
     - unzip /tmp/armbinary.zip
     - mv bin/* ./
     - rm -rf bin
+  artifacts_test_script:  # Other CI systems depend on all files being present
+    - ls -la
```

Comment:
What does this `ls -la` is this for debugging.

Reply:
Yes it shows the files that are actually present, since they are renamed in the build task.  It makes it easier to diagnose any test failure, as opposed to requiring a developer to manually download the artifact zip to check.

Reply:
SGTM thanks for the explanation :)